### PR TITLE
napi port

### PR DESCRIPTION
**Pull request by wijiler**
I think this is what you wanted?

### CODE REVIEW

File: rusties/cpu-count/Cargo.toml [hunk 1]
```diff
@@ -4,13 +4,12 @@ version = "0.1.0"
...
 crate-type = ["cdylib"]
 
 [dependencies]
+napi = "2"
+napi-derive = { version = "2",default-features = true}
 
-[dependencies.neon]
-version = "0.10"
-default-features = false
-features = ["napi-6"]
```
Comment (Izocel) [inline rusties/cpu-count/Cargo.toml:16]: I think neon already uses napi ?

File: rusties/cpu-count/package.json [hunk 2]
```diff
@@ -1,18 +1,16 @@
 {
-  "name": "cpu-count",
-  "version": "0.1.0",
-  "description": "",
+  "name": "cpucount",
+  "version": "1.0.0",
+  "description": "cpucount",
   "main": "index.node",
-  "scripts": {
```
Comment (Izocel) [inline rusties/cpu-count/package.json:6]: this was building a index.node

File: rusties/cpu-count/src/lib.rs [hunk 3]
```diff
@@ -1,11 +1,5 @@
-use neon::prelude::*;
```
Comment (Izocel) [inline rusties/cpu-count/src/lib.rs:1]: Why removing the 2 fonction that i wanted to export ?

**Review by Izocel — CHANGES_REQUESTED** [review]
- We cannot get rid of neon-preludes
- Why Removing Cargo.lock  and the pacakge.json ? (never do this its too early in the project)
- Were's the node and import of the created module into main.ts ?

File: rusties/cpu-count/Cargo.toml (hunk 1 above)
Comment (wijiler) [reply rusties/cpu-count/Cargo.toml:16]: didnt notice that

**Comment by wijiler** [comment]
the node I deleted as I was just cleaning up and It must've gotten caught in the crossfire

**Comment by wijiler** [comment]
will make changes later,but right now I have to focus on one of my projects

**Comment by Izocel** [comment]
No problem i'll close it then you could do another PR if you wish

**Pull request closed without merging**